src/popup/locales/en.json

```diff
@@ -309,12 +312,19 @@
     },
     "send": {
       "sendTitle": "Send funds",
+      "sendBtc": "Send Bitcoin",
       "reviewTitle": "Review transaction",
       "successTitle": "Transaction successfully submitted",
       "recipientLabel": "Recipient",
       "recipientPlaceholder": "Enter public address or .chain name",
       "recipientPlaceholderUrl": "Enter public address, .chain name or URL",
-      "payload": "Payload"
+      "recipientPlaceholderBtc": "Enter Bitcoin public address",
+      "transactionSpeed": "Transaction speed",
+      "transactionWillBeCompleted": "Transaction will be completed in ≈{time}.",
+      "payload": "Payload",
+      "fast": "Fast",
+      "medium": "Medium",
+      "slow": "Slow"
```

Comment:
I think this might be used not only in the send modal so I suggest to move it to the `common`. Eg.: `common.transferSpeed.<speed>`.